data/json/items/containers/containers.json

```diff
@@ -2405,6 +2405,34 @@
     ],
     "melee_damage": { "bash": 1 }
   },
+  {
+    "id": "dvd_case",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "DVD case" },
+    "description": "A protective container designed to hold a Digital Versatile Disc (or DVD, for short).  It is made of plastic and, before the Cataclysm, was used to package and store DVDs, providing protection against scratches, dust, and damage.",
+    "weight": "100 g",
+    "volume": "184 ml",
+    "longest_side": "19 cm",
+    "price": 1000,
+    "price_postapoc": 10,
+    "material": [ "plastic" ],
+    "symbol": ";",
+    "color": "black",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "1 ml",
+        "max_contains_weight": "16 g",
+        "max_item_length": "1 mm",
```

Comment:
I get that this is still in progress, but I will note this down regardless. Unless you add DVDs proportionally sized with mosquitoes, a 1mm length restriction will probably result in nothing being able to be put within the case. Though you have detailed item restrictions, there still needs to be enough length room for the item to actually fit. Also, you might want to consider making the case a holster, i.e., only capable of holding a single item. The current size already means that, likely, only one disc would fit anyhow, but it future-proofs against any potential dimension changes that might happen in the future.

Reply:
Hello Doc. It is a honour to have you here at this PR today. Yes, I agree on that, and I don't see a reason why it can't be a holster. But, if I am not mistaken, wouldn't it make the DVD case item have a text like "sheath/draw weapon or tool?"